examples/GainPlugin/PluginProcessor.h

```diff
@@ -15,7 +15,7 @@
 //==============================================================================
 /**
 */
-class GainPluginAudioProcessor  : public AudioProcessor
+class GainPluginAudioProcessor  : public AudioProcessor, public Timer

```

Comment:
Nice stuff @ilionic !

Super minor comment from me here. Would inherit privately from `juce::Timer` here.

Reply:
Ah ok, updated. I actually had it all namespaced since I am using Juce 6 and removed it to adapt to Juce 5. 
Btw, perhaps worth considering updating example to Juce6?

Reply:
Ahh sorry, I meant private inheritance rather than namespacing!

So like:

```
class GainPluginAudioProcessor  : public AudioProcessor, private Timer
```

Reply:
But yeah we should probably update the code accordingly here!

Reply:
@JoshMarler do you want me to update to Juce6 part of this PR or separate? There are not many changes, few places need to be prefixed with juce namespaces.

Reply:
Ahhh no don't worry about that right now. Lets keep this commit as a separate thing as you've kept it really nice and clean. Small concise commits FTW

Reply:
Ok updated to private Timer. Removed juce namespacing prefix for later when updating to Juce 6 to keep things consistent.